crawl-ref/source/tileview.cc

```diff
@@ -1392,6 +1393,12 @@ void tile_apply_properties(const coord_def &gc, packed_cell &cell)
             cell.bg |= TILE_FLAG_TRAV_EXCL;
     }
 
+    if(you.rampaging() && adjacent(you.pos(), gc) && Options.show_rampage_indicator
```

Comment:
Trivia: this is wrong! `adjacent` returns true even if `you.pos() == gc` - we need `grid_distance(you.pos(), gc) == 1`.